corehq/apps/api/urls.py

```diff
@@ -182,6 +181,7 @@ def versioned_apis(api_list):
     fixtures.v0_1.LookupTableItemResource.get_urlpattern('v1'),
     fixtures.v0_6.LookupTableItemResource.get_urlpattern('v2'),
     v0_5.NavigationEventAuditResource.get_urlpattern('v1'),
+    v1_0.CommCareAnalyticsUserResource.get_urlpattern('v1'),
```

Comment:
Oooh, very nice!  Another thing you may consider if it makes resonates with you is to put the resource in the app containing the model it serves.  So the users resources in the `users` app, the groups resource in the `groups` app, etc.  We have a [few](https://github.com/dimagi/commcare-hq/blob/136859de6d201c2c2728e0ec3c6eb324c1eee412/corehq/apps/api/urls.py#L54-L59) that operate this way already, and I've been thinking about pushing us to that model eventually, though I certainly haven't established a consensus around it yet.

My thinking is that it's more in line with how Django apps are generally used - one app per "thing", with models, views, tests, forms, fixtures, migrations, etc all in that folder, so related code is close together.  Though in this case I'm not certain there's a very natural home for `CommCareAnalyticsUserResource` - maybe `users`?  Also fine to punt on that question